components/esp_system/Kconfig

```
@@ -138,6 +138,13 @@ menu "ESP System Settings"
                 for the data part (above the splitting address). The memory protection is effective
                 on all access through the IRAM0 and DRAM0 buses.
 
+        config ESP_SYSTEM_PMP_LP_CORE_RESERVE_MEM_EXECUTABLE
+            bool "Make LP core reserved memory executable in HP core"
```

nitpick, should it be
```suggestion
            bool "Make LP core reserved memory executable from HP core"
```

---

Typo fixed, thanks!